### DISCUSSION TITLE
how to set a customize url to start code-server

### DISCUSSION THREAD

**Discussion by jaffe-fly** [coder/code-server#4381] [Q&A] [answered]
i want to visit url with my customize  like ```192.268.34.16:8080/predict```

if dont use nginx ,is there any method?

and how to use ```--proxy-domain``` 

when i start with ```--proxy-domain=123.2345```  ,i cant into ```http://192.168.30.24:8080.123.2345```or  ```http://192.168.30.24:8080/123.2345```  it will  404 error

**Comment by jsjoeio** [comment]
Gooood question! I haven't used `--proxy-domain` before. Any ideas @code-asher ?

**Comment by code-asher** [comment, marked as answer]
--proxy-domain is for when you are using code-server's built-in proxy to
view ports via a sub-domain (for example `8080.my-code-server.com` to
view port 8080).  It only works with domains, not IP addresses, and it
has nothing to do with a base path.

If you want a base path like `[ip address]/predict` then you will have
to use a reverse proxy like Caddy, NGINX, etc.

**Reply by jaffe-fly** [reply]
jupyter notebook can custom you own path，this is a good function,,you can realise it
